relayer/app.go

```diff
@@ -549,40 +568,23 @@ func (a *App) validatePassphrase(envPassphrase string) error {
 func (a *App) Start(ctx context.Context, tunnelIDs []uint64) error {
 	a.Log.Info("Starting tunnel relayer")
 
-	isSyncTunnelsAllowed := false
-
 	// query tunnels
-	var tunnels []bandtypes.Tunnel
-	if len(tunnelIDs) == 0 {
-		var err error
-		tunnels, err = a.BandClient.GetTunnels(ctx)
-		isSyncTunnelsAllowed = true
-		if err != nil {
-			return err
-		}
-	} else {
-		tunnels = make([]bandtypes.Tunnel, 0, len(tunnelIDs))
-		for _, tunnelID := range tunnelIDs {
-			tunnel, err := a.BandClient.GetTunnel(ctx, tunnelID)
-			if err != nil {
-				return err
-			}
-			tunnels = append(tunnels, *tunnel)
-		}
+	tunnels, err := a.getTunnels(ctx, tunnelIDs)
+	if err != nil {
+		a.Log.Error("Cannot get tunnels", zap.Error(err))
 	}
 
 	if len(tunnels) == 0 {
 		a.Log.Error("No tunnel ID provided")
 		return fmt.Errorf("no tunnel ID provided")
 	}
```

Comment:
it also can happen when run with just `start` command but there's no tunnel on chain?

Reply:
for that case, we should not return error.